lib/web_ui/lib/src/engine/configuration.dart

```diff
@@ -320,7 +333,22 @@ external JsFlutterConfiguration? get _jsConfiguration;
 /// The JS bindings for the object that's set as `window.flutterConfiguration`.
 @JS()
 @staticInterop
-class JsFlutterConfiguration {}
+@anonymous
+class JsFlutterConfiguration {
+  external factory JsFlutterConfiguration({
+    JSString? assetBase,
+    JSString? canvasKitBaseUrl,
+    JSString? canvasKitVariant,
+    JSBoolean? canvasKitForceCpuOnly,
+    JSNumber? canvasKitMaximumSurfaces,
+    JSBoolean? debugShowSemanticsNodes,
+    DomElement? hostElement,
+    JSString? nonce,
+    JSString? renderer,
+    JSBoolean? useColorEmoji,
+    JSBoolean? loadManifestFontsBeforeAppMain,
+  });
```

Comment:
Please, do not expose a Dart constructor for this object. We already have a way to override JS Configuration values from Dart (without messing with the config set by the test runner): `debugOverrideJsConfiguration`.